posthog/api/property_definition.py

```diff
@@ -97,25 +99,24 @@ def get_queryset(self):
         }
 
         if use_entreprise_taxonomy:
+            # Select all columns except `tags` specifically so that queryset doesn't try to fetch one-to-many tags
             return EnterprisePropertyDefinition.objects.raw(
                 f"""
-                SELECT posthog_propertydefinition.*,
-                       ee_enterprisepropertydefinition.*, 
+                SELECT propertydefinition_ptr_id, description, updated_at, updated_by_id, id, name, is_numerical, volume_30_day, query_usage_30_day, team_id, property_type, property_type_format,
                        {event_property_field} AS is_event_property
-                FROM posthog_propertydefinition
-                LEFT JOIN ee_enterprisepropertydefinition ON ee_enterprisepropertydefinition.propertydefinition_ptr_id=posthog_propertydefinition.id
-                WHERE posthog_propertydefinition.team_id = %(team_id)s AND name NOT IN %(excluded_properties)s {name_filter} {search_query}
-                GROUP BY posthog_propertydefinition.id, ee_enterprisepropertydefinition.propertydefinition_ptr_id
+                FROM ee_enterprisepropertydefinition
+                FULL OUTER JOIN posthog_propertydefinition ON posthog_propertydefinition.id=ee_enterprisepropertydefinition.propertydefinition_ptr_id
+                WHERE team_id = %(team_id)s AND name NOT IN %(excluded_properties)s {name_filter} {search_query}
```

Comment:
I'm trying to figure out what practically changed in this query after the select statement, but am struggling to see it...

Also same thought for the "except tags" aspect of the select statement. Could we do this programatically, so we don't have to remember to update this when more fields are added to the model.

Reply:
I was equally confused reading this and was hoping to get the green light to standardize this query. I think this is just tech debt from having two separate ppl write these queries. I ran these queries in the SQL console and they both seem to return the same results.